signac/synced_collections/errors.py

```diff
@@ -4,10 +4,7 @@
 """Errors raised by synced collections."""
 
 
-from ..errors import Error
-
-
-class BufferException(Error):
+class BufferException(Exception):
     """An exception occured in buffered mode."""
```

Comment:
We need to clarify that this is not just an exception "in" buffered mode, since that sounds like _any_ exception raised during a buffered context will be re-raised by this class. I'm not sure if my suggestion solves it... but I tried to match the style of the other errors' docstrings.
```suggestion
    """Raised when an exception related to buffering occurs."""
```